.github/workflows/golangci-lint.yml

```diff
@@ -44,11 +45,21 @@ jobs:
           filters: |
             docs:
               - 'docs/**/*'
+              - '*.md'
+            go:
+              - '**/*.go'
```

Comment:
Shoud also add `tools.go` and `main.go`.

Reply:
Yep. Addressed.